guides/source/layouts_and_rendering.md

```diff
@@ -103,32 +103,6 @@ In most cases, the `ActionController::Base#render` method does the heavy lifting
 
 TIP: If you want to see the exact results of a call to `render` without needing to inspect it in a browser, you can call `render_to_string`. This method takes exactly the same options as `render`, but it returns a string instead of sending a response back to the browser.
 
-#### Rendering Nothing
```

Comment:
http://guides.rubyonrails.org/layouts_and_rendering.html#using-head-to-build-header-only-responses